### DISCUSSION TITLE
Icon Names

### DISCUSSION THREAD

**Discussion by PeterPetersen** [FortAwesome/Font-Awesome#20279] [General]
How do i know that the mug-saucer icon from Font-Awesome is the faCoffee-Icon in FortAwesome?

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

I may not have got the question, but `coffee` is an alias of `mug-saucer` now I guess and it can be used with both names, because we tend to use the actual object instead of its abstract representation

You can know that it is the same icon because of alias tags

https://fontawesome.com/icons/mug-saucer?f=classic&s=solid

![image](https://github.com/FortAwesome/Font-Awesome/assets/556268/ac2a0192-ca88-4c98-9973-a2841921244e)

Note: This answer may not be accurate, let's ask @robmadole, it looks like a recent change: https://blog.fontawesome.com/icon-naming-conventions/

**Reply by PeterPetersen** [reply]
hi and thanks for rapid reply! i did not know the green links are alias. I tried a few icons and it works! Thank You!!!!

**Comment by robmadole** [comment]
Yep! That's correct.

**Comment by sensibleworld** [comment]
And for anyone interested, here's why we made some name changes: https://blog.fontawesome.com/icon-naming-conventions/
